src/main/java/com/vmware/vipclient/i18n/base/cache/CacheItem.java

```diff
@@ -12,11 +12,13 @@ public CacheItem() {
 		
 	}
 	
-	public CacheItem (Map<String, String> dataMap, final Map<String, Object> cacheProps) {
+	public CacheItem (Map<String, String> dataMap, String etag, long timestamp, Long maxAgeMillis) {
 		super();
 		if (dataMap != null) 
```

Comment:
Suggest to move this judge into addCachedData method, because It's necessary all the time.